### ISSUE TITLE
npx create-tina-app@latest throwing error

### ISSUE THREAD

**Issue by fergiemcdowall** [tinacms/tinacms#4744]
### Describe the issue that you're seeing. Any Loom videos or screenshots usually help a lot!

Running `create-tina-app` seems to throw errors no matter what combination of options I choose.

~~Either this error:~~ (**EDIT:** actually this works when `yarn` is globally stored)

```
fergus@Ferguss-MacBook-Pro-2 tina-test % npx create-tina-app@latest
✔ Which package manager would you like to use? › yarn
✔ What is your project named? … my-tina-app
✔ What starter code would you like to use? › ⭐ NextJS starter
Downloading files from repo tinacms/tina-cloud-starter. This might take a moment.
Installing packages. This might take a couple of minutes.

node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: spawn yarn ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:292:12)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn yarn',
  path: 'yarn',
  spawnargs: [ 'install' ]
}

Node.js v20.12.2
```

Or this one:

```
fergus@Ferguss-MacBook-Pro-2 tina-test % npx create-tina-app@latest
✔ Which package manager would you like to use? › npm
✔ What is your project named? … my-tina-app
✔ What starter code would you like to use? › ⭐ NextJS starter
Downloading files from repo tinacms/tina-cloud-starter. This might take a moment.
Installing packages. This might take a couple of minutes.

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @tinacms/starter@0.1.0
npm ERR! Found: eslint@9.9.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^9.8.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.23.0 || ^8.0.0" from eslint-config-next@14.2.4
npm ERR! node_modules/eslint-config-next
npm ERR!   dev eslint-config-next@"14.2.4" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/fergus/.npm/_logs/2024-08-23T08_35_30_282Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/fergus/.npm/_logs/2024-08-23T08_35_30_282Z-debug-0.log
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".] {
  code: 'ERR_UNHANDLED_REJECTION'
} 

Node.js v20.12.2
```

### Reproduction

https://tina.io/docs/introduction/using-starter/?_gl=1*nx1s1j*_up*MQ..*_ga*MTUzMDc0NTE0NC4xNzI0NDAwNTQw*_ga_9R1HHZC8HX*MTcyNDQwMDUzOS4xLjAuMTcyNDQwMDUzOS4wLjAuMTc5NjQ1Mjc2

### Steps to reproduce

As per the steps here -> https://tina.io/docs/introduction/using-starter/?_gl=1*nx1s1j*_up*MQ..*_ga*MTUzMDc0NTE0NC4xNzI0NDAwNTQw*_ga_9R1HHZC8HX*MTcyNDQwMDUzOS4xLjAuMTcyNDQwMDUzOS4wLjAuMTc5NjQ1Mjc2

### System Info

  System:
    OS: macOS 14.4.1
    CPU: (10) arm64 Apple M2 Pro
    Memory: 1.77 GB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.12.2 - ~/.nvm/versions/node/v20.12.2/bin/node
    npm: 10.5.0 - ~/.nvm/versions/node/v20.12.2/bin/npm
  Browsers:
    Chrome: 127.0.6533.122
    Edge: 128.0.2739.42
    Safari: 17.4.1

(sidenote: not relevant here, but my main browser is Firefox)

### Validations

- [x] Follow our [Code of Conduct](https://github.com/tinacms/tinacms/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [docs](https://tina.io/docs).
- [x] Check that there isn't [already an issue](https://github.com/tinacms/tinacms/issues) that reports the same bug.

**Comment by fergiemcdowall** [comment]
Can be fixed by installing `yarn` globally (`npm install -g yarn`)

**Comment by fergiemcdowall** [comment]
Reopening, because installer still fails when `npm` is selected as package manager

**Comment by despino-streaver** [comment]
> Reopening, because installer still fails when `npm` is selected as package manager

Exactly. The solution I found for this is, as you may suppose, to downgrade the eslint version listed in the package.json file in order to install the latest one which doesn't conflicts with eslint-config-next dependencies (i.e. setting `"eslint": "^8.0.0"`).

**Comment by fergiemcdowall** [comment]
I wonder if eslint@8.0.0 should be the default or if it would mess up something else?

**Comment by kldavis4** [comment]
I believe this was due to an upgrade that occurred here https://github.com/tinacms/tina-cloud-starter/pull/535 

@JackDevAU you can reproduce this just by cloning main `tina-cloud-starter` and running `npm i`

I believe nextjs uses an older version of eslint (8.x), so this is probably fixed just by downgrading eslint to that [version](https://github.com/vercel/next.js/blob/v14.2.11/package.json#L136) or just keep the dependency on `eslint-config-next` and remove `eslint`

**Comment by brookjeynes-ssw** [comment]
Done - This has been resolved by downgrading eslint to v8.0.0 and will be available within the next release.
